src/main/java/io/github/jbellis/brokk/gui/GitLogTab.java

```diff
@@ -972,10 +971,7 @@ private void handleBranchPopup(MouseEvent e) {
             int selectedRow = branchTable.getSelectedRow();
             if (selectedRow != -1) {
                 String branchDisplay = (String) branchTableModel.getValueAt(selectedRow, 1);
-                if (branchDisplay.startsWith("Local: ")) {
```

Comment:
`Local:` prefix seems to be a leftover from a refactoring in the past; it is not present in the logic related to displaying branches anywhere else, so it can be safely removed if I see correctly.